scene/scene_string_names.h

```diff
@@ -83,6 +83,7 @@ class SceneStringNames {
 
 	StringName pre_sort_children;
 	StringName sort_children;
+	StringName request_fit_child_in_rect;
```

Comment:
This isn't used anywhere except once when it's emitted so it doesn't really justify adding it here IMO